### PR TITLE
Add the Remote coordinator for TPUVM pod

### PR DESCRIPTION
**Pull request by JackCaoG**
couple gotcha
1. remote coordinator needs to have `pt/xla 1.8.1` installed
2. remote coordinator needs to have `gcloud alpha` installed

### CODE REVIEW

File: test/test_xla_dist.py [hunk 1]
```diff
@@ -704,6 +721,121 @@ def test_healthy_cluster(self):
...
+        noop_compute_service)
+
+    tpu_map = {
+        'fake-pod': {
+            'state':
+                'READY',
+            'health':
+                'HEALTHY',
+            'runtime_version':
+                'v2-nightly',
```
Comment (jysohn23) [inline test/test_xla_dist.py:736]: This reminds me, let's add on our user guide that users must not use `v2-.*-pod` TPU runtime versions as that one also starts TF server up and our XRT server will get "/dev/accel[0-4]  already in use" errors.

File: test/test_xla_dist.py [hunk 2]
```diff
@@ -704,6 +721,121 @@ def test_healthy_cluster(self):
...
+            'accelerator_type':
+                'v3-32',
+            'api_version':
+                'V2_ALPHA1',
+            'network_endpoints': [{
+                'ipAddress':
+                    f'10.1.0.{index}',
+                'port':
+                    '8470',
+                'greenVmSelflink':
```
Comment (jysohn23) [inline test/test_xla_dist.py:746]: This should be removed. Internal only.

**Review by jysohn23 — CHANGES_REQUESTED** [review]
Thanks @JackCaoG!

File: test/test_xla_dist.py (hunk 2 above)
Comment (JackCaoG) [reply test/test_xla_dist.py:746]: Hmm, for running script on tpuvm case, we actually query the `greenVmSelflink` to get the instance name to build the client worker. Should that also be changed?

File: test/test_xla_dist.py (hunk 1 above)
Comment (JackCaoG) [reply test/test_xla_dist.py:736]: yea, I will mention that.

File: test/test_xla_dist.py (hunk 2 above)
Comment (jysohn23) [reply test/test_xla_dist.py:746]: Yes as that's an internal field. Syncing offline.

**Review by jysohn23 — APPROVED** [review]

**Pull request merged**